grakn-test/src/test/java/ai/grakn/test/graql/template/TemplateParserTest.java

```diff
@@ -58,7 +71,7 @@ public void multiValueOneLineTest(){
         assertParseEquals(template, data, expected);
     }
 
-    @Test(expected = RuntimeException.class)
+    @Test(expected = GraqlTemplateParsingException.class)
```

Comment:
Looking into using `ExpectedException` instead. It lets you check the contents of the error message as well. It's a more accurate check.